src/seraphis_wallet/key_container.cpp

```diff
@@ -0,0 +1,223 @@
+// paired header
+#include "key_container.h"
+
+// local headers
+#include "crypto/chacha.h"
+#include "misc_log_ex.h"
+#include "ringct/rctOps.h"
+#include "seraphis_core/jamtis_core_utils.h"
+#include "seraphis_core/sp_core_enote_utils.h"
+#include "jamtis_keys.h"
+#include "seraphis_wallet/encrypted_file.h"
+
+// standard headers
+
+#undef MONERO_DEFAULT_LOG_CATEGORY
+#define MONERO_DEFAULT_LOG_CATEGORY "seraphis_wallet"
+
+namespace seraphis_wallet
+{
+//-------------------------------------------------------------------------------------------------------------------
+// INTERNAL
+static jamtis_keys ser_keys_to_jamtis(ser_jamtis_keys const &keys)
```

Comment:
As far as I remember we never properly discussed [this issue](https://github.com/seraphis-migration/wallet3/issues/57) I wrote back in summer whether we really use the Seraphis library scheme of *pairs* of classes, one "normal" and one `ser_` class, for serialization of wallet classes. Maybe now it's the time to come back to this?

Reply:
I don't think they should be here